tests/modules/nf-core/proteinortho/main.nf

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PROTEINORTHO } from '../../../../modules/nf-core/proteinortho/main.nf'
+
+workflow test_proteinortho {
+
+    file_a = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
+    file_a.copyTo("${workDir}/tmp/prot1.fasta")
+    file_a = file("${workDir}/tmp/prot1.fasta", checkIfExists:true)
+    file_b = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
+   
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        [ 
+            file_a,
```

Comment:
```suggestion
            file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true).renameTo("prot1.fasta")
```

Reply:
Although I'm not sure why you are renaming it, the module is staging the file into a subdirectory anyway